apps/bfd-model/bfd-model-rif/src/main/resources/db/migration/V60__Alter_beneficiaries_bene_id_numeric.sql

```diff
@@ -0,0 +1,35 @@
+-- ALTER_BENEFICIARIES_BENE_ID_NUMERIC.SQL
+-- add a column to beneficiaries table; an ALTER TABLE by definition
+-- creates an 'access exclusive' lock on the table.
+ALTER TABLE public.beneficiaries ADD bene_id_numeric bigint;
+
+-- create a database trigger which will be used to imbue a bigint value
+-- into the bene_id_numeric column  during a table record INSERT operation.
+-- The trigger is defined as a pre-insert trigger, which means that prior to
+-- new record being written, the database engine will execute the trigger,
+-- which in turn, fills the bene_id_numeric with a value based on a 'cast'
+-- of the bene_id (varchar).
+
+-- HSQL always gets created/initialized from scratch so no need to drop trigger
+${logic.hsql-only} CREATE TRIGGER beneficiaries_insert_trigger BEFORE INSERT ON public.beneficiaries
```

Comment:
Out of curiosity, why is the trigger needed for HSQL?  Will there be some future test cases that depend on it?

Reply:
good point....while certainly not really needed, it's there in case someone asked the reciprocal question...why don't you have it. also, if someone has an IT where they are inserting a record into the beneficiaries table, then we'd want to at least make things work like postgres.